spec/features/visit_my_vacancies_page_spec.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.feature 'My Vacancies Page', type: :feature do
+  context 'with a logged user' do
+    let(:user) { create(:user) }
+
+    scenario 'show the vacancies list' do
+      create(:vacancy)
```

Comment:
I think you should create a vacancy for another company just to test if it is listing jobs only for this company.

```
create(:vacancy)
create(:vacancy, company: create(:company))
```

something like this

Reply:
You didn't fix this. I think it is a good idea to create a job for another company too and check if it is still listing one job only